openghg_defs/data/site_info.json

```diff
@@ -978,6 +997,22 @@
     }
   },
   "GAT": {
+    "ICOS": {
+      "latitude": 53.0657,
+      "longitude": 11.4429,
+      "height_station_masl": 70.0,
+      "long_name": "Gartow, Germany",
+      "height": ["132m"],
+      "height_name": ["132magl"]
+    },
+    "ICOS": {
```

Comment:
Similiarly here - did these need to be separate ICOS blocks? Or did I misunderstand the intended structure?

Reply:
Nice spot, I have merged the two together